soc/arm/nxp_imx/rt/soc.c

```diff
@@ -143,6 +147,8 @@ static ALWAYS_INLINE void clock_init(void)
 	CLOCK_InitUsb1Pll(&usb1PllConfig); /* Configure USB1 PLL to 480M */
 #endif
 #ifdef CONFIG_INIT_ENET_PLL
+	/* DeInit Enet PLL. */
+	CLOCK_DeinitEnetPll();
```

Comment:
because we are adjust the PLL settings in the initEnetPLL and then reenable it, so I thin kthis would be safer to deinit it before that, and this is what SDK does.

Reply:
This is not done for other PLL's, seems not needed.

Reply:
ok, remove